### DISCUSSION TITLE
twitter çağrılarının yapay zeka(open.ai) ile filtrelenmesi

### DISCUSSION THREAD

**Discussion by ahmetcagatay** [acikkaynak/deprem-yardim-projesi#51] [Ideas]
Openai ile twitter'dan çekilen verileri kolayca temizleyip parse edebiliriz. Verilerin temizlenmesi katmanı süreci daha kolay halledilebilir böylece. (Zekeriya Mulbay'ın fikridir. biz üzerinde çalışıyoruz belki üzerine koyarsınız diye sizinle de paylaşayım dedim.)
<img width="437" alt="image" src="https://user-images.githubusercontent.com/42320353/218214394-d3ee2a3e-7ffb-4cd6-8862-dfb6ab263aee.png">
![image](https://user-images.githubusercontent.com/42320353/218214075-d6aeefb6-bb25-4e28-a98b-1aeb0e4dc068.png)
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/42320353/218216539-1207dc87-d721-4259-865d-5041d2aad086.png">

**Comment by zekeriyamulbay** [comment]
Prompt ciktisini ekliyorum
<img width="800" alt="Screenshot 2023-02-11 at 08 29 44" src="https://user-images.githubusercontent.com/67113841/218248618-a4ba0b78-4ef0-4700-a3ab-2a935fd4841e.png">

**Comment by zekeriyamulbay** [comment]
Bu da prompt (ciktiyi JSON olarak isleyebilirsiniz)

{{tweet}}

Yukaridaki tweet'e bakarak asagidaki JSON'i doldur:
{"isim": (tweet'i atan kisinin ismi),
"tel": (tweet'teki telefon numarasi),
"adres": (tweet'teki Google Maps uyumlu tam adres),
"sehir": (tweet'teki Google Maps uyumlusehir adi),
"mahalle": (tweet'teki Google Maps uyumlumahalle adi),
"cadde": (tweet'teki Google Maps uyumlu cadde adi),
"koy": (tweet'teki Google Maps uyumlu koy adi),
"kasaba": (tweet'teki Google Maps uyumlu kasaba adi),
"ihtiyaclar":(ihtiyaclari virgul ile ayirarak ekle)
}

**Reply by zekeriyamulbay** [reply]
Kisi sayisini da ekledik:

{{tweet}}

Yukaridaki tweet'e bakarak asagidaki JSON'i doldur:
{"isim": (tweet'i atan kisinin ismi),
"tel": (tweet'teki telefon numarasi),
"adres": (tweet'teki Google Maps uyumlu tam adres),
"sehir": (tweet'teki Google Maps uyumlusehir adi),
"mahalle": (tweet'teki Google Maps uyumlumahalle adi),
"cadde": (tweet'teki Google Maps uyumlu cadde adi),
"koy": (tweet'teki Google Maps uyumlu koy adi),
"kasaba": (tweet'teki Google Maps uyumlu kasaba adi),
"sayi": (varsa tweet'teki kisi sayisi),
"ihtiyaclar":(ihtiyaclari virgul ile ayirarak ekle)
}

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Hali hazırda bunu yapmaktayız.
